domains/lophoctinhthuong.is-local.org.json

```diff
@@ -11,7 +11,8 @@
 
     "record": {
         "MX": ["smtp.google.com"],
-        "TXT": ["google-site-verification=qPjtcM6ve8YVnBcO7VeMFsOm9pkhCXoYTQhMs0qwo-M"]
+        "TXT": ["google-site-verification=qPjtcM6ve8YVnBcO7VeMFsOm9pkhCXoYTQhMs0qwo-M"],
+        "CNAME": ["hosykynam0619ch2008-umego.github.io"]
```

Comment:
If you are going to use a CNAME record, it must be alone per DNS rules. CNAME records also cannot be in an array.